gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -123,7 +123,7 @@
             <message key="import.illegal" value="Use JUnit 4-style (org.junit.*) test classes and assertions instead of JUnit 3 (junit.framework.*)."/>
         </module>
         <module name="IllegalImport"> <!-- Java Coding Guidelines: Import the canonical package -->
-            <property name="illegalPkgs" value="org.elasticsearch.common.base, jersey.repackaged.com.google.common, com.google.api.client.repackaged, org.apache.hadoop.thirdparty.guava, com.clearspring.analytics.util, org.spark_project.guava"/>
+            <property name="illegalPkgs" value="org.elasticsearch.common.base, jersey.repackaged.com.google.common, com.google.api.client.repackaged, org.apache.hadoop.thirdparty.guava, com.clearspring.analytics.util, org.spark_project.guava, avro.shaded.com.google.common"/>
```

Comment:
we should probably add .*shaded.*, .*thirdparty.* as illegal imports!

Reply:
`.*repackaged.*`, then, too

Reply:
Do we want that to be done in this PR then?

Reply:
I actually tried to use regexes to ban stuff for gradle internal, but it didn't work with out old version of checkstyle.  This might have changed now that we've upgraded checkstyle... can you screenshot this rule working?

Reply:
Yes, I had to bump the checkstyle version. I used 7.8.2. I didn't realize that the repo had upgraded to 8.12.